### ISSUE TITLE
Drop-down menu functionality

### ISSUE THREAD

**Issue by joyfuldoctor** [Simplero/Roadmap#379]
Hello 
We are trying to create a 'drop-down' list of tabs that appears when we hover on our main menu tabs. 
So we would be able to see a 'drop-down' list of tabs for pages that are available within the main menu tab when we hover over the main menu tab.
Cheers
Tom @ Team Joy

**Comment by nicolabird** [comment]
Yes please, I would like that too - just like a normal website navigation bar.

**Comment by JeppeThy** [comment]
I miss that function too!!

**Comment by emilyannpeterson** [comment]
Plus 1 for a drop-down menu option in membership sites!
